src/pages/library/_viewId.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <v-app-bar fixed flat dense class="second-toolbar">
```

Comment:
This breaks the current way these toolbar work, as they no longer stay at the top when scrolling, which is imo a lot less user-friendly (You have to scroll back up, sometimes through hundreds of items, to change a filter or something...)